src/components/atoms/Text/Text.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import styles from './Text.scss';
+
+const Text = ({ children, className, tag, theme, ...props }) => {
+    const TagName = tag;
+    const classNames = cx({
+        [className]: className,
+        [styles.headline]: theme === 'headline',
+        [styles.subheadline]: theme === 'subheadline',
+        [styles.body]: theme === 'body',
+    });
+    return (
+        <TagName {...props} className={classNames}>
+            {children}
+        </TagName>
+    );
+};
+
+Text.propTypes = {
+    children: PropTypes.node.isRequired,
+    className: PropTypes.string,
+    tag: PropTypes.string,
+    theme: PropTypes.string,
```

Comment:
This should probably be a oneOf PropType to throw a warning when someone passes a bad theme

Reply:
Good call. Updated: https://github.com/MadeInHaus/react-redux-webpack-starter/pull/34/commits/c523efb98ced812945754e885fbedfebf2c49eff